packages/devtools-launchpad/bin/development-server.js

```diff
@@ -83,6 +85,21 @@ function handleNetworkRequest(req, res) {
   }
 }
 
+function handleLaunchRequest(req, res) {
+  const browser = req.body.browser;
+  const location = "https://devtools-html.github.io/debugger-examples/";
+
+  process.env.PATH += `:${__dirname}`;
+  if (browser == "Firefox") {
+    ps.spawn("firefox-driver.js", ["--start", "--location", `${location}`])
```

Comment:
I think you could probably call it directly :)

it exports a `start` function 
https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-launchpad/bin/firefox-driver.js#L105